main.py

```diff
@@ -347,6 +347,69 @@ def make(self):
                 # totaltime+=timer.tick()
                 if event.type == pygame.QUIT:
                     crashed = True
+                # keypress check
+                else:            	
+		            if keyinit == 0 and :
```

Comment:
`and :` followed with no condition will return an error and activity won't start.

Reply:
Thanks, went unnoticed :smile:

Reply:
Found some important bugs which need to be solved before release
![image](https://user-images.githubusercontent.com/48695438/83785648-4b7ed880-a69a-11ea-8887-b99aad3711a7.png)

Reply:
How do I reproduce? Didn't get this error before!

Reply:
Make the stick longer than the screen width by holding down the key

Reply:
We get this error at this line https://github.com/sugarlabs/stick-hero-activity/blob/52c16c851313c1acce157fe3898c4d286356f4be/main.py#L831
 because heropointer in L831 is initialised to 0 before. so time will also become 0 if this doesnt become true https://github.com/sugarlabs/stick-hero-activity/blob/52c16c851313c1acce157fe3898c4d286356f4be/main.py#L576

Reply:
Ok that was my fault; :/

I found this bug on master as well however
![image](https://user-images.githubusercontent.com/48695438/83797639-a2d97480-a6ab-11ea-8eed-5d0566edf586.png)

Reply:
@Saumya-Mishra9129, it was my fault in the new code intended to fix another bug. Thats the reason; Normally the values for `heropointer` should be in the closed range `[800, 900]`, but for some reason, in the new code I added, in order to get the event poll, I had by mistakedly, resetted the variable of `heropointer` to 0 always making it crash on the second and third try; It should be fixed now.

Reply:
Thanks @srevinsaju , I wonder now why we can see it with https://github.com/sugarlabs/stick-hero-activity/pull/29#issuecomment-664121256.